app/controllers/concerns/bbb_server.rb

```diff
@@ -54,6 +54,9 @@ def join_path(room, name, options = {}, uid = nil)
     join_opts = {}
     join_opts[:userID] = uid if uid
     join_opts[:join_via_html5] = true
+    options.select { |a, _| a.to_s.starts_with?("userdata-bbb") }.each do |k, v|
+      join_opts[k] = v
+    end
```

Comment:
Can you use filter here instead of select?

Reply:
Can we also use descriptive variable names instead of `a` `k` and `v`?

Reply:
Ah thanks for checking

Reply:
I have make changes for this please review it